src/main/kotlin/no/nav/familie/ef/mottak/service/SøknadServiceImpl.kt

```diff
@@ -36,14 +38,17 @@ class SøknadServiceImpl(private val soknadRepository: SoknadRepository,
     override fun sendTilSak(søknadId: String) {
         val soknad: Soknad = soknadRepository.findByIdOrNull(søknadId) ?: error("")
 
-                if (soknad.dokumenttype == DOKUMENTTYPE_OVERGANGSSTØNAD) {
-                    val sak: Sak = SakMapper.toSak(soknad)
-                    søknadClient.sendTilSak(sak)
-
-                } else {
-                    val skjemasak: Skjemasak = SakMapper.toSkjemasak(soknad)
-                    søknadClient.sendTilSak(skjemasak)
-                }
+        if (soknad.dokumenttype == DOKUMENTTYPE_OVERGANGSSTØNAD) {
+            if (featureToggleService.isEnabled("familie.ef.mottak.send-til-sak")) {
+                val sak: Sak = SakMapper.toSak(soknad)
+                søknadClient.sendTilSak(sak)
+            } else {
+                logger.info("Sender ikke søknad til sak, feature familie.ef.mottak.send-til-sak er skrudd av i Unleash ")
```

Comment:
Dette syns jeg blir litt rart. Vi har en task at det skal bli sendt til sak.
Er det ikke bedre hvis vi kan returnere en ny status i task-grensesnittet, slik det er mulig å markere de som ikke er håndtert "riktigt"?

Eks sette status til **IGNORERT** eller noe, så vi kan rekjøre de enkelt hvis vi trenger?

Reply:
Og hvis ikke så bør vi logge hvilken ID på saken det gjelder

Reply:
Tja, men husk at vi har også Arbeidssøkerskjema som vi skal håndtere - og vi skiller i utgangspunktet ikke på disse to ved opprettelse av tasks. Da må vi gjøre denne sjekken over her for å se om vi skal opprette oppgave eller ikke. 

Ignorert er forsåvidt en interessant idé, men da må du inn i felles-koden og legge inn logikk der ettersom dette ikke er noe vi styrer med herfra. 

Min tanke er at _ALLE_ søknader vi mottar blir ignorert - følgelig kan vi si at vi skal opprette tasks for ALLE søknader den dagen vi ønsker å sende disse til sak. Men skjema blir sendt allikevel.

Reply:
Jeg foreslår å flytte koden til HentSaksnummerFraJoarkTask.onCompletion, og ikke gjøre noe hvis ikke send-til-sakl er enabled. Da blir hentSaksnummerFraJoark siste task og vi opprettet ikke noen sendTilSakTask og vi slipper å lyve om at vi har sendt til sak når vi ikke har gjort det. Søknad og skjema kan behandles likt inntil videre da ingen av dem har noe i sak å gjøre for øyeblikket.